src/edge/simulator.ts

```diff
@@ -114,7 +114,13 @@ export class Simulator {
             TelemetryClient.sendEvent(`${telemetryName}.${type}`);
 
             const installResult = await this.autoInstallSimulator(outputChannel);
-            TelemetryClient.sendEvent(`${telemetryName}.${type}.${InstallReturn[installResult.resultType]}`, { error: installResult.errMsg });
+
+            if (installResult.errMsg) {
+                TelemetryClient.sendErrorEvent(`${telemetryName}.${type}.${InstallReturn[installResult.resultType]}`, { error: installResult.errMsg });
+            } else {
+                TelemetryClient.sendEvent(`${telemetryName}.${type}.${InstallReturn[installResult.resultType]}`);
```

Comment:
there is no property in this scenario? So nothing will be sent?

Reply:
Just send an event with the name such as "simulatorUpdated.install.Success"